hoverxref-intersphinx.rst

```diff
@@ -0,0 +1,114 @@
+
+This extension is an essential part
+of the work we are doing to improve interoperability of documentation in general,
+and scientific documentation in particular,
+thanks to the `CZI grant </czi-grant-announcement>`_ we received last year.
+
+.. figure:: img/sphinx-hoverxref-math.png
+   :width: 80%
+   :align: center
+   :alt: sphinx-hoverxref displaying a tooltip including an equation
+   :target: _images/sphinx-hoverxref-math.png
+
+   sphinx-hoverxref displaying a tooltip including an equation
+
+.. _sphinx-hoverxref: https://sphinx-hoverxref.readthedocs.io/
+
+Embedding documentation fragments
+---------------------------------
+
+Technical documentation is full of cross-references (or *xrefs*):
+hyperlinks to related parts of the documentation (or even to an external project)
+that help contextualize the information.
+`Sphinx provides powerful tools for creating
+xrefs <https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html>`_,
+which can be used to link other parts of the documentation,
+and in the case of software projects, also to specific objects of the API.
+
+However, jumping through several links while reading some documentation
+can be distracting, since opening them might need either a context switch
+or keeping a new browser tab for a later moment.
+
+Here is where `sphinx-hoverxref`_ comes to the rescue:
+it scans the cross-references and attaches floating windows to them
+(either *tooltips* or *modal* dialogues)
+that embed the contents of the page the xrefs point to.
+This way, when the user is interested in a particular topic,
+they can hover the mouse over the link
+and display the floating window
+instead of opening the link.
+
+Moreover, the floating windows created by `sphinx-hoverxref`_
+also work if the embedded content requires a particular extension to be rendered.
+This makes it possible to embed content that includes mathematical equations, for example.
+
+Support for ``intersphinx``
+---------------------------
+
+Version 0.6b1 of `sphinx-hoverxref`_ introduces opt-in support for Intersphinx linking.
+`sphinx.ext.intersphinx <https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html>`_
+is a Sphinx extension
+that generates automatic links to the documentation of objects in other projects,
+and now it's possible to show floating windows on them too.
+
+.. figure:: img/sphinx-hoverxref-intersphinx.gif
+   :width: 80%
+   :align: center
+   :alt: sphinx-hoverxref displaying a modal window embedding an external project
+   :target: _images/sphinx-hoverxref-intersphinx.gif
+
+   sphinx-hoverxref displaying a modal window embedding an external project
+
+To enable it, the user needs to specify
+which projects from the `intersphinx_mapping <https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#confval-intersphinx_mapping>`_
+will be scanned by ``sphinx-hoverxref``, using a new configuration value:
+
+.. code-block:: python
+
+   hoverxref_intersphinx = [
```

Comment:
I still feel like this name is not great, and should be more explicit. I feel like this code example actually needs a comment to explain _what_ the content of this list is:

```suggestion
   # Values of the intersphinx_mapping keys that you want to enable hoverxref on
   hoverxref_intersphinx = [
```

Reply:
Good catch 👍🏽